### PR TITLE
Move tunneling CLI changes from master to dev mode

### PR DESCRIPTION
**Pull request by deka108**
Tunneling CLI will be moved to development mode. Since Command and Execution SDKs were added as part of the tunneling changes, these will be removed as well.

### CODE REVIEW

File: databricks_cli/sdk/version.py [hunk 1]
```diff
@@ -21,8 +21,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-API_VERSION = '2.0'
-OLD_API_VERSION = '1.2'
+API_VERSION='2.0'
 
 # Available API versions
-API_VERSIONS = ['2.0', '2.1']
+API_VERSIONS=['2.0', '2.1']
```
Comment (jaipreet-s) [inline databricks_cli/sdk/version.py:27]: We can leave these as there should be a space between '='

File: lint.sh [hunk 2]
```diff
@@ -2,15 +2,9 @@
...
-PY_VERSION=${1:-3}
-
 FWDIR="$(cd "`dirname $0`"; pwd)"
 cd "$FWDIR"
 
-if [[ "$PY_VERSION" -eq 3 ]]; then
-  prospector --profile "$FWDIR/prospector.yaml"
-else
-  prospector --profile "$FWDIR/prospector-2.yaml"
-fi
```
Comment (jaipreet-s) [inline lint.sh:14]: Again, I'm OK leaving these in as these help isolate py2 vs py3 dependencies

File: tox.ini [hunk 3]
```diff
@@ -8,8 +8,6 @@ deps =
   py27: -rtox-requirements-2.txt
   py36: -rtox-requirements.txt
 commands =
-	py36: pytest tests --cov=./
-	py27: pytest tests --ignore=tests/tunnel --ignore=tests/commands --ignore="tests/sdk/test_v1_service.py" --cov=./
-	py36: ./lint.sh
-	py27: ./lint.sh 2
+	pytest tests --cov=./
+	./lint.sh
```
Comment (jaipreet-s) [inline tox.ini:12]: Same comment here re: py2 vs py3

**Review by jaipreet-s — CHANGES_REQUESTED** [review]

File: databricks_cli/sdk/api_client.py [hunk 4]
```diff
@@ -147,16 +144,14 @@ def perform_query(self, method, path, data={}, headers=None, files=None, version
...
-            return resp
         return resp.json()
 
 
     def get_url(self, path, version=None):
         if version:
             return self.url + version + path
         elif self.jobs_api_version and path and path.startswith('/jobs'):
-            return self.url + self.jobs_api_version + path
+            return self.url + self.jobs_api_version + path
```
Comment (joveyuan-db) [inline databricks_cli/sdk/api_client.py:154]: If this doesn't make a re-revert difficult, we can probably remove the extra whitespace here.

**Review by joveyuan-db — APPROVED** [review]

**Review by jaipreet-s — APPROVED** [review]

**Pull request merged**